app/templates/frameworks/_framework_actions.html

```diff
@@ -1,5 +1,5 @@
 {% if framework.status in ['standstill', 'live'] and application_made and not countersigned_agreement_file %}
-  {% if not framework.frameworkAgreementVersion %}
+  {% if not framework.frameworkAgreementVersion or not supplier_is_on_framework %}
```

Comment:
Yep makes sense.  By the time I worked out what the condition should actually be my mind was pretty much melted so I just went with it.